packages/launch/src/focused-launch/summary/focused-launch-summary-item/style.scss

```diff
@@ -0,0 +1,79 @@
+@import '~@automattic/typography/styles/variables';
+@import '~@automattic/onboarding/styles/variables';
+
+$accent-blue: #117ac9;
+
+.focused-launch-summary__item {
+	border: 2px solid var( --studio-gray-5 );
+	padding: 10px 14px;
+	min-height: 64px;
+	/* stylelint-disable-next-line scales/radii */
+	border-radius: 5px;
```

Comment:
From the design, it looks like the border radius is `4px`. Also, we should understand why `4px` is still not in the [list of allowed border-radii](https://github.com/Automattic/wp-calypso/blob/12df4ecc07993c51c549616de6a1c93ce7d95077/.stylelintrc#L6)